sympy/stats/error_prop.py

```diff
@@ -0,0 +1,44 @@
+"""Tools for arithmetic error propogation."""
```

Comment:
This line should be routinely included.
``from __future__ import print_function, division``